From 194f520f578e4d5d3a549ba83d44497f1124ab64 Mon Sep 17 00:00:00 2001
From: Shadi Ammouri <shadi@marvell.com>
Date: Tue, 2 Feb 2016 21:16:59 +0200
Subject: [PATCH 0773/1240] mppbus: cp: rename CP mppbus entries to cp-mpp-bus

	This prevents override of AP mpp-bus entries by CP mpp-bus
	entries.

Change-Id: Id0155f6b34f41cbdee60b23abf371387f906556e
Signed-off-by: Shadi Ammouri <shadi@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/27156
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm/dts/armada-70x0-dev-info.dtsi | 28 ++++++++++++++--------------
 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git a/arch/arm/dts/armada-70x0-dev-info.dtsi b/arch/arm/dts/armada-70x0-dev-info.dtsi
index b361f09..d2075af 100644
--- a/arch/arm/dts/armada-70x0-dev-info.dtsi
+++ b/arch/arm/dts/armada-70x0-dev-info.dtsi
@@ -87,84 +87,84 @@
 		};
 	};
 	mpp-buses {
-		mpp-bus0 {
+		cp-mpp-bus0 {
 			compatible = "marvell,mvebu-mpp-bus";
 			bus-name = "spi1";
 			bank-name = "cp110";
 			pin-count = <4>;
 			pins = <13 3  14 3  15 3  16 3>;
 		};
-		mpp-bus1 {
+		cp-mpp-bus1 {
 			compatible = "marvell,mvebu-mpp-bus";
 			bus-name = "i2c";
 			bank-name = "cp110";
 			pin-count = <2>;
 			pins = <37 2  38 2>;
 		};
-		mpp-bus2 {
+		cp-mpp-bus2 {
 			compatible = "marvell,mvebu-mpp-bus";
 			bus-name = "uart0";
 			bank-name = "cp110";
 			pin-count = <2>;
 			pins = <29 0xA  30 0xA>;
 		};
-		mpp-bus3 {
+		cp-mpp-bus3 {
 			compatible = "marvell,mvebu-mpp-bus";
 			bus-name = "uart1";
 			bank-name = "cp110";
 			pin-count = <2>;
 			pins = <40 7  41 7>;
 		};
-		mpp-bus4 {
+		cp-mpp-bus4 {
 			compatible = "marvell,mvebu-mpp-bus";
 			bus-name = "sd";
 			bank-name = "cp110";
 			pin-count = <7>;
 			pins = <56 0xE  57 0xE  58 0xE 59 0xE  60 0xE  61 0xE  62 0xE>;
 		};
-		mpp-bus5 {
+		cp-mpp-bus5 {
 			compatible = "marvell,mvebu-mpp-bus";
 			bus-name = "tdm";
 			bank-name = "cp110";
 			pin-count = <12>;
 			pins = <0 4  1 4  2 4  3 4  4 4  5 4  6 4  7 4  8 4  9 4  10 4  11 4>;
 		};
-		mpp-bus6 {
+		cp-mpp-bus6 {
 			compatible = "marvell,mvebu-mpp-bus";
 			bus-name = "rgmii0";
 			bank-name = "cp110";
 			pin-count = <12>;
 			pins = <0 3  1 3  2 3  3 3  4 3  5 3  6 3  7 3  8 3  9 3  10 3  11 3>;
 		};
-		mpp-bus7 {
+		cp-mpp-bus7 {
 			compatible = "marvell,mvebu-mpp-bus";
 			bus-name = "rgmii1";
 			bank-name = "cp110";
 			pin-count = <11>;
 			pins = <45 2  46 1  47 1  48 1  49 1  50 1  51 1  52 1  53 1  54 1  55 1>;
 		};
-		mpp-bus8 {
+		cp-mpp-bus8 {
 			compatible = "marvell,mvebu-mpp-bus";
 			bus-name = "nand";
 			bank-name = "cp110";
-			pin-count = <13>;
-			pins = <13 2  17 1  18 1  19 1  20 1  21 1  22 1  23 1  24 1  25 1  26 1  27 1 31 1>;
+			pin-count = <14>;
+			pins = <13 2 15 1 16 1 17 1 18 1 19 1 20 1 21 1 22 1 23 1 24 1 25 1 26 1 27 1>;
 		};
-		mpp-bus9 {
+		cp-mpp-bus9 {
 			compatible = "marvell,mvebu-mpp-bus";
 			bus-name = "sata0";
 			bank-name = "cp110";
 			pin-count = <1>;
 			pins = <13 6>;
 		};
-		mpp-bus10 {
+		cp-mpp-bus10 {
 			compatible = "marvell,mvebu-mpp-bus";
 			bus-name = "sata1";
 			bank-name = "cp110";
 			pin-count = <1>;
 			pins = <28 9>;
 		};
-		mpp-bus11 {
+		cp-mpp-bus11 {
 			compatible = "marvell,mvebu-mpp-bus";
 			bus-name = "smi";
 			bank-name = "cp110";
-- 
1.9.1

